test/unit/math/prim/fun/mdivide_left_tri_low_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include <stan/math/prim.hpp>
+#include <test/unit/math/prim/fun/expect_matrix_eq.hpp>
+#include <gtest/gtest.h>
+
+#define EXPECT_MATRIX_NEAR(A, B, DELTA) \
+  for (int i = 0; i < A.size(); i++)    \
+    EXPECT_NEAR(A(i), B(i), DELTA);
+
+TEST(MathMatrixPrim, mdivide_left_tri_low_val) {
```

Comment:
Thanks for filling in unrelated missing tests!

There should be a size 1 test as well as size 2 and size 0, just to check that boundary conditions are working.